Source/WebKit/Configurations/WebContentService.xcconfig

```diff
@@ -30,6 +30,12 @@ PRODUCT_NAME_Development = com.apple.WebKit.WebContent.Development;
 PRODUCT_NAME_CaptivePortal = com.apple.WebKit.WebContent.CaptivePortal;
 
 PRODUCT_BUNDLE_IDENTIFIER = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphone*] = $(PRODUCT_NAME)Service;
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphone*17.0*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphone*17.1*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphone*17.2*] = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphone*17.3*] = $(PRODUCT_NAME);
```

Comment:
Could these just be `sdk=iphone*17*]` instead?

Reply:
These are intentionally _not_ matching the OS versions where we're using extensions.